oam/install_scripts/post-install

```diff
@@ -206,24 +206,22 @@ fi
 #setup MariaDB Columnstore system logging
 if [ $user = "root" ]; then
 	$installdir/bin/syslogSetup.sh install > /tmp/syslog_install.log 2>&1
-	chmod 777 -R /dev/shm
 	rm -f /etc/default/columnstore
 else
-	$SUDO chmod 777 /tmp
+	$SUDO chmod 755 /tmp
 	$SUDO rm -fr /tmp/* > /dev/null 2>&1
 	$SUDO $installdir/bin/syslogSetup.sh --installdir=$installdir install > /tmp/syslog_install.log 2>&1
 	$SUDO chown $user:$user $installdir/etc/Columnstore.xml
-	$SUDO chmod -R 777 /dev/shm
+	$SUDO chmod -R 755 /dev/shm
```

Comment:
Since this is a root owned directory (or 777 with sticky depending on Linux release) non-root install will no longer be able to write to /dev/shm with this change.